libs/entitlement/asm-provider/build.gradle

```diff
@@ -14,9 +14,11 @@ dependencies {
   compileOnly project(':libs:core')
   compileOnly project(':libs:logging')
   implementation 'org.ow2.asm:asm:9.7.1'
+  implementation 'org.ow2.asm:asm-util:9.7.1'
+  implementation 'org.ow2.asm:asm-tree:9.7.1'
+  implementation 'org.ow2.asm:asm-analysis:9.7.1'
   testImplementation project(":test:framework")
   testImplementation project(":libs:entitlement:bridge")
-  testImplementation 'org.ow2.asm:asm-util:9.7.1'
```

Comment:
Does it work if you remove this? I was under the impression that, in ElasticSearch, `implementation` does not imply `testImplementation` so you need both.

Reply:
Good question, I assumed it did. 
Tests run though...

Reply:
Works for me. 🤷